src/app/hero/hero.component.html

```diff
@@ -1,4 +1,4 @@
-<div [class]="'hero ' + (webp ? 'webp-bg' : 'nowebp-bg')">
+<div [ngStyle]="{'background-image': 'url(' + campaign.bannerUri + ')'}">
```

Comment:
@tbg-jasonfung  don't we still need `hero` as a fixed/static class?

Reply:
Ahh oops!